playwright/src/utilities/utilities.ts

```diff
@@ -2,19 +2,23 @@ import { ElementHandle, Page } from '@playwright/test'
 
 async function closeExtras(page: Page, retries = 5): Promise<void> {
   try {
-    await page.click('text=Got it!', { timeout: 500 })
-  } catch (error) {}
-  try {
-    await page.click('lower-sticky-note-content > .put-away', { timeout: 500 })
-  } catch (error) {}
-  try {
-    await page.click('._pi_closeButton', { timeout: 500 })
-  } catch (error) {}
-  try {
-    await page.click('.ReactModalPortal .put-away', { timeout: 500 })
+    await page.locator('text=Got it!').click()
+    // await page.click('text=Got it!', { timeout: 500 })
   } catch (error) {}
+  // try {
+  //   await page.click('lower-sticky-note-content > .put-away', { timeout: 500 })
+  // } catch (error) {}
+  // try {
+  //   await page.click('._pi_closeButton', { timeout: 500 })
+  // } catch (error) {}
+  // try {
+  //   await page.click('.ReactModalPortal .put-away', { timeout: 500 })
+  // } catch (error) {}
+  // const extras = await page.isVisible(
+  //   '.cookie-notice button, lower-sticky-note-content > .put-away, ._pi_closeButton, .ReactModalPortal .put-away',
+  // )
```

Comment:
@gregfitch can you please confirm if the commented lines (8-19) are Kinetic stuff

Reply:
1: Web, 2: Web, 3: REx, 4: Web